### PR TITLE
Aux secondary exposure control

### PR DESCRIPTION
**Pull request by mattalvarado**
Add secondary exposure controls for the aux color camera in the ROS driver. This was tested on a S27 with v5.12 firmware.

### CODE REVIEW

File: multisense_ros/src/reconfigure.cpp [hunk 1]
```diff
@@ -93,6 +94,12 @@ Reconfigure::Reconfigure(Channel* driver,
         roi_supported_ = true;
     }
 
+    if (system::DeviceInfo::HARDWARE_REV_MULTISENSE_C6S2_S27 == deviceInfo.hardwareRevision ||
+        system::DeviceInfo::HARDWARE_REV_MULTISENSE_S30 == deviceInfo.hardwareRevision)
```
Comment (psmithcrl) [inline multisense_ros/src/reconfigure.cpp:98]: The new monocam will need this as well.

Comment (mattalvarado) [reply multisense_ros/src/reconfigure.cpp:98]: Can we add that once we have hardware in hand?

**Review by jackg0 — COMMENTED** [review]
Should S27s default to auto_exposure_thresh of 0.95 in multisense.cfg?

**Comment by mattalvarado** [comment]
> Should S27s default to auto_exposure_thresh of 0.95 in multisense.cfg?

This should be fixed on master

**Review by jackg0 — APPROVED** [review]

**Pull request merged**